### PR TITLE
Initial commit BC001 - BU01

### PR DESCRIPTION
**Pull request by pjanck**
- example file
- ids file
- documentation

### CODE REVIEW

File: tests/BU01/Dataset/README.md
Comment (evandroAlfieri) [inline tests/BU01/Dataset/README.md:1]: what about adding the requirements for the property set + properties to the IDS? Something like
```
            <ids:applicability>
                <ids:entity>
                    <ids:name>
                        <ids:simpleValue>IFCBUILDING</ids:simpleValue>
                    </ids:name>
                </ids:entity>
            </ids:applicability>
            <ids:requirements>
                <ids:property measure="IFCLABEL">
                    <ids:propertySet>
                        <ids:simpleValue>SBB-CFF-FFS</ids:simpleValue>
                    </ids:propertySet>
                    <ids:name>
                        <ids:simpleValue>Status</ids:simpleValue>
                    </ids:name>
                </ids:property>
            </ids:requirements>
```
Or even going further constraining the values, where possible.

**Review by evandroAlfieri — CHANGES_REQUESTED** [review]

File: tests/BU01/Dataset/README.md
Comment (pjanck) [reply tests/BU01/Dataset/README.md:1]: This is not a *requirement* of the test case (see [tests/BU01/readme.md](https://github.com/buildingSMART/IFC4.x-IF/pull/37/files#diff-d428a6592276c570f000550005913d5746d7602c8e57755a6de9bd4359c8cb39)). It is only a property of the provided reference dataset. The implementers are free to include it, however absence of these properties does not invalidate their dataset.

**Review by evandroAlfieri — APPROVED** [review]

**Review by BenzclyZhang — APPROVED** [review]

**Comment by evandroAlfieri** [comment]
@BenzclyZhang just made a small PR for a quick change as branch to the rail repo. Please have a look and if it's ok update this PR, so I can merge. Thanks

**Comment by evandroAlfieri** [comment]
> @BenzclyZhang just made a small PR for a quick change as branch to the rail repo. Please have a look and if it's ok update this PR, so I can merge. Thanks

Hi @BenzclyZhang, can you review and approve the small change I suggested - so that this test case can be published? Thanks
cc: @pjanck @apinzenoehler

**Comment by pjanck** [comment]
> @BenzclyZhang just made a small PR for a quick change as branch to the rail repo

Simile to #38 - I can't seem to be able to find the PR. Can you please provide a link?

**Comment by evandroAlfieri** [comment]
Forgot to PR from the patch branch, sorry. Here: https://github.com/bSI-RailwayRoom/IFC4.x-IF/pull/6

**Comment by pjanck** [comment]
@evandroAlfieri This should be resolved now and ready for merging.

**Pull request merged**